packages/patterns-react/.env.example

```diff
@@ -1,17 +0,0 @@
-PATTERNS_STORYBOOK_SOURCEMAPS=<Boolean to turn on/off sourcemaps in storybook>
-
-#Feature Flags
-DDS_FLAGS_ALL=<Boolean flag to turn on all feature flags>
-
-DDS_CARD_ARRAY=<Boolean flag to turn on/off the CardArray component>
-DDS_CARDS_WITHOUT_IMAGES=<Boolean flag to turn on/off the CardsWithoutImages component>
-DDS_LEADSPACE=<Boolean flag to turn on/off the lead space pattern>
-DDS_LEADSPACE_CENTERED=<Boolean flag to turn on/off the LeadSpaceCentered component>
-DDS_LISTSECTION=<Boolean flag to turn on/off the ListSection component>
-DDS_PICTOGRAM_ARRAY=<Boolean flag to turn on/off the PictogramArray component>
-DDS_SIMPLELONGFORM=<Boolean flag to turn on/off the SimpleLongForm component>
-DDS_TOC=<Boolean flag to turn on/off the TableOfContents component>
-DDS_USECASES=<Boolean flag to turn on/off the UseCases component>
-
-# Carbon Expressive Theme
-PATTERNS_STORYBOOK_CARBON_EXPRESSIVE=<Boolean to turn on/off the carbon expressive theme in storybook>
```

Comment:
@RaphaelAmadeu why was this file deleted?

Reply:
I've restored it on the last commit, It was deleted by mistake